ui/lib/config-ui/addon/components/messages/page/list.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { getOwner } from '@ember/application';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+
+/**
+ * @module Page::MessagesList
+ * Page::MessagesList components are used to display breadcrumb links. This is component will be replaced when HDS system is incorporated
+ *
+ * @example
+ * ```js
+ * <Page::MessagesList @messages={{this.messages}}  />
+ * ```
+ * @param {array} messages - array message objects
+ */
+
+export default class MessagesList extends Component {
+  @service store;
+
+  get mountPoint() {
+    // mountPoint tells transition where to start. In this case, mountPoint will always be vault.cluster.config-ui
+    return getOwner(this).mountPoint;
+  }
+
+  // callback from HDS pagination to set the queryParams page
+  get paginationQueryParams() {
+    return (page) => {
+      return {
+        page,
+      };
+    };
+  }
+
+  @task
+  *deleteMessage(message) {
```

Comment:
Curious why this is a concurrency task and not just async/await. Unless we'er using one of their loading states (ex: this.deleteMessages.isRunning) or drop functions (ex: `}).drop()`), I would say use async/await.

Reply:
I'll most likely add a loading icon for delete messages